helper/cli.c

```diff
@@ -274,6 +374,215 @@ static int cmd_call_odp_pktio_print(struct cli_def *cli,
 	return CLI_OK;
 }
 
+static int cmd_call_odp_pktio_stats_print(struct cli_def *cli,
+					  const char *command ODP_UNUSED,
+					  char *argv[], int argc)
+{
+	if (check_num_args(cli, argc, 1))
+		return CLI_ERROR;
+
+	odp_pktio_t hdl = odp_pktio_lookup(argv[0]);
+
+	if (hdl == ODP_PKTIO_INVALID) {
+		cli_error(cli, "%% Name not found.");
+		return CLI_ERROR;
+	}
```

Comment:
Repeated in each pktio command, so could be made a separate function. Especially if  we end up splitting `odp_pktio_queue_stats_print()` and `odp_pktio_event_queue_stats_print()`.

Reply:
Refactored this a bit, should be slightly less repetitive now. v2.